deployment/cfn/utils/cfn.py

```diff
@@ -42,30 +44,42 @@ def get_availability_zones(aws_profile):
     return [AvailabilityZone(az) for az in conn.get_all_zones()]
 
 
+def get_availability_zones(aws_profile, zones):
+    """Helper function that returns availability zones for a region
+
+    Returns:
+      (list of AvailabilityZone): List of availability zones for a given
+                                  EC2 region
+    """
+    conn = boto.connect_ec2(profile_name=aws_profile)
+    return [AvailabilityZone(az) for az in conn.get_all_zones(zones=zones)]
+
+
 def get_subnet_cidr_block():
     """Generator to generate unique CIDR block subnets"""
     current = 0
     high = 255
     while current <= high:
-        yield '10.0.%s.0/24' % current
+        yield "10.0.%s.0/24" % current
         current += 1
 
 
-def get_recent_ami(aws_profile, ami_name, owner="self", executable_by="self"):
-    conn = boto.connect_ec2(profile_name=aws_profile)
-    filters = {'name': ami_name}
+def get_recent_ami(
```

Comment:
Pulling in updates to this method from Cicero.

Reply:
I'll go through and make corrections to things. I had been using https://github.com/python/black (without tuning any of its parameters), although I agree the method signature you posted is much more pretty.